relay-event-schema/src/protocol/contexts/device.rs

```diff
@@ -161,6 +162,22 @@ pub struct DeviceContext {
     /// Whether location support is available on the device.
     pub supports_location_service: Annotated<bool>,
 
+    /// Width of the screen in pixels.
+    #[metastructure(pii = "maybe")]
+    pub screen_width_pixels: Annotated<u64>,
```

Comment:
I thought about this, and I was like, maybe some people want to scrape those fields, since they might not want to share that information with sentry (e.g., a new device with a new resolution is being secretly tested), maybe the UUID is the only thing but also that can be delicate for some users. We can definitely omit it, idk if there is a performance overhead to this.

Reply:
I'd rather not mark them PII. They are not, and none of the other fields are marked as such.